MaybeGame/GameMaster.cs

```diff
@@ -1,10 +1,63 @@
-﻿namespace MaybeGame
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MaybeGame
 {
-    partial class Program
+    partial class MainProgram
     {
         public class GameMaster
         {
+            private List<Player> _players { get; } = new();
+
+            public void PlayerDataDump()
+            {
+                foreach (var p in _players)
+                    p.DataDump();
+            }
+
+            public void Start()
+            {
+                Console.WriteLine("\nMenu:\n" +
```

Comment:
Start method does way too much. Export this to a method **WriteGameRules()**

Reply:
Noted and exported